cns/NetworkContainerContract.go

```diff
@@ -406,6 +412,14 @@ type PodIpInfo struct {
 	PodIPConfig                     IPSubnet
 	NetworkContainerPrimaryIPConfig IPConfiguration
 	HostPrimaryIPInfo               HostIPInfo
+	// NICType defines whether NIC is Default or Secondary
+	NICType string
+	// MacAddress of (secondary) interface
+	MacAddress string
+	// SkipDefaultRoutes is true if default routes should not be added on interface
+	SkipDefaultRoutes bool
+	// Routes to configure on interface
+	Routes []Route
```

Comment:
should we pass InterfaceName as well.. today containerd passed interface name for infranic and for secondary interface, we are hardcoding it in cni.. interfaceName can be empty for infraNic